src/core/server/saved_objects/import/create_saved_objects.test.ts

```diff
@@ -699,7 +721,56 @@ describe('#createSavedObjects', () => {
           },
         },
       ];
-      await testVegaVisualizationsWithDataSources({
+      await testVegaTimelineVisualizationsWithDataSources({
+        objects,
+        expectedFilteredObjects,
+        dataSourceId: 'some-datasource-id',
+        dataSourceTitle: 'dataSourceName',
+      });
+    });
+  });
+
+  describe('with a data source for timeline saved objects', () => {
+    test('can attach a data source name to the timeline expression', async () => {
```

Comment:
Same deal. Could you add some tests to cover multiple `.opensearch()` functions within an expression?

Reply:
+1

Reply:
added